components/x-gift-article/src/v2/SocialShareButtons.jsx

```diff
@@ -0,0 +1,96 @@
+import { h } from '@financial-times/x-engine'
+import { ShareType } from '../lib/constants'
+
+export const SocialShareButtons = ({ actions, mailtoUrl, mobileShareLinks, shareType }) => {
+	const onClickHandler = (event) => {
+		switch (shareType) {
+			case ShareType.gift:
+				actions.emailGiftUrl(event)
+				break
+			case ShareType.enterprise:
+				actions.emailEnterpriseUrl(event)
+				break
+			case ShareType.nonGift:
+				actions.emailNonGiftUrl(event)
+				break
+			default:
+		}
+	}
+
+	return (
+		<div>
+			<h4 className="share-article-dialog__share-via-socials-title">Or share via</h4>
+			<div data-o-component="o-share" className="o-share">
+				<ul className="share-article-dialog__share-via-socials-wrapper">
+					<li className="o-share__action">
+						<a
+							className="o-share__icon o-share__icon--twitter"
+							href={mobileShareLinks.twitter}
+							rel="noopener"
```

Comment:
add `data-trackable="twitter"` and the same on each link to simplify tracking

Reply:
see [1ca0928](https://github.com/Financial-Times/x-dash/pull/719/commits/1ca09280293213b17dc9375c1601b24e90a980e3)